Assets/MixedRealityToolkit.Services/InputSimulation/MixedRealityInputSimulationProfile.cs

```diff
@@ -113,17 +113,17 @@ public class MixedRealityInputSimulationProfile : BaseMixedRealityProfile
 
         [Header("Hand Gesture Settings")]
         [SerializeField]
-        private SimulatedHandPose.GestureId defaultHandGesture = SimulatedHandPose.GestureId.Open;
-        public SimulatedHandPose.GestureId DefaultHandGesture => defaultHandGesture;
+        private ArticulatedHandPose.GestureId defaultHandGesture = ArticulatedHandPose.GestureId.Open;
```

Comment:
is this a breaking change? meaning was SimulatedHandPose ever publicly available?

Reply:
_Technically_ yes since it was/is a public class, but highly unlikely that any external developer actually used it.